psi4/driver/procrouting/proc.py

```diff
@@ -1924,6 +1930,9 @@ def set_cholesky_from(corl_type):
            "mp3":     {"wfn_type": "DF-OMP3",     "orb_opt": "FALSE", "nat_orbs": "FALSE", "corl_type": core.get_global_option("MP_TYPE") if core.has_global_option_changed("MP_TYPE") else "DF"},
           "omp3":     {"wfn_type": "DF-OMP3",     "orb_opt": "TRUE",  "nat_orbs": "FALSE", "corl_type": core.get_global_option("MP_TYPE")},
 
+         "remp2":     {"wfn_type": "DF-OREMP",    "orb_opt": "FALSE", "nat_orbs": "FALSE", "corl_type": core.get_global_option("CC_TYPE")},
```

Comment:
good question, I asked SB this, too. `MP_TYPE` looks more likely at first, but remp is a hybrid of perturbation theory, and at one end of a dial, it equals LCCD, and in fact REMP2 reuses large portions of LCCD code. So since LCCD is governed by `CC_TYPE`, and SB preferred it, going with `CC_TYPE` for remp.

Reply:
That makes sense, thanks!